scitt/check_operation_status.py

```diff
@@ -0,0 +1,46 @@
+""" Module for checking when a statement has been anchored in the append-only ledger """
```

Comment:
I am unsure whether we want to obfuscate the get operation ID endpoint in a sample poller or not.

On the one hand it stops people on the getting started spamming it till it is ready.

On the other hand it moves them away from understanding the SCITT API we have.

Reply:
agreed. see https://github.com/datatrails/datatrails-scitt-samples/pull/4#discussion_r1412465303
Lets discuss on Monday. It's the balance between the Quickstart and the SCITT API docs.